app/core/config.py

```
@@ -45,6 +45,10 @@ class Settings(BaseSettings):
     SPRING_CALLBACK_MAX_RETRIES: int = 2
     ANALYSIS_CALLBACK_INTERNAL_TOKEN: str | None = None
     
+    # Git clone
+    GIT_CLONE_ROOT_DIR: str = "/tmp/secause-analysis"
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/secause-analysis"

(S108)